src/content/docs/en/reference/container-reference.mdx

```diff
@@ -40,25 +40,30 @@ Enables rendering components using [HTML streaming](/en/guides/server-side-rende
 
 #### `renderers` option
 
-**Type:** `AstroRenderer[]`
+**Type:** `SSRLoadedRenderer[]`
 
-A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must provide an object stating the integration package `name`, as well as both its client and server rendering script
+A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must use function `getContainerRenderer` that can be imported from the renderer library.
 
-The following example provides the necessary object to render an Astro component that renders a React component: 
+When using `vite` (`vitest`, Astro integrations, etc.), the renderers should be loaded by using the function `loadRenderers`, from the virtual module `astro:container`. Outside `vite`, you'll have to load the renderers manually.
+
+The following example provides the necessary object to render an Astro component that renders a React component and a Svelte component: 
 
 ```js
+import { getContainerRenderer as reactContainerRenderer } from "@astrojs/react";
+import { getContainerRenderer as svelteContainerRenderer } from "@astrojs/svelte";
+import { loadRenderers } from "astro:container";
+
+const renderers = loadRenderers([reactContainerRenderer(), svelteContainerRenderer()]);
 const container = await experimental_AstroContainer.create({
-    renderers: [
-      {
-	      name: "@astrojs/react",
-	      clientEntrypoint: "@astrojs/react/client.js",
-	      serverEntrypoint: "@astrojs/react/server.js"
-      }
-    ]
+    renderers
 })
 const result = await container.renderToString(ReactWrapper);
 ```
 
+:::note
+The function `getContainerRenderer` is only available for renderers inside the `@astrojs/` npm org. Renderers outside this org will provide a similar/same function.
+:::
+
```

Comment:
```suggestion
```
We try to avoid notes when we can, especially if we can work the text into a normal paragraph. I tried to do that above. See what you think!